src/main/java/com/fontys/api/configurations/WebConfig.java

```diff
@@ -0,0 +1,16 @@
+package com.fontys.api.configurations;
+
+import org.springframework.context.annotation.Configuration;
+import org.springframework.web.servlet.config.annotation.CorsRegistry;
+import org.springframework.web.servlet.config.annotation.EnableWebMvc;
+import org.springframework.web.servlet.config.annotation.WebMvcConfigurer;
+
+@Configuration
+@EnableWebMvc
+class WebConfig implements WebMvcConfigurer {
+
+    @Override
+    public void addCorsMappings(CorsRegistry registry) {
+        registry.addMapping("/**");
```

Comment:
Accepting everything is maybe not the safest thing.

Reply:
I have currently added the deployed frontend